pegasus/sites.v3/code.org/views/dance_teacher_resources.haml

```diff
@@ -1,20 +1,29 @@
 %link{:rel=>'stylesheet', :type=>'text/css', :href=>'/css/dance-landing/dance-teacher-resources.css'}
 
 :ruby
+  locale = I18n.locale.to_s.downcase
+  translated_languages = ["es-mx", "es-es", "fr-fr", "ko-ko", "tr-tr", "zh-tw"]
```

Comment:
@tanyaparker  Yep, this is a temporary solution.  Elijah has a ToDo to determine a solution for this. (deployment.rb)

Reply:
> Curriculum builder should have the logic to show you the translated version if it exists, otherwise it falls back to English.

Note that the fallback cannot be as simple as this, since curriculumbuilder is literally just an S3 bucket, with no ability to be aware of its own content like this

Reply:
Agreed that this isn't the most sustainable long-term solution. To mitigate some of the issue of having to remember when a new locale is added, is there a good place in curriculum builder or the i18n pipeline for a reminder comment that this needs to be updated?